backend/satellite_tools/repo_plugins/yum_dnf_src.py

```diff
@@ -202,17 +204,15 @@ def setup_repo(self, repo, no_mirrors, ca_cert_file, client_cert_file, client_ke
         self.dnfbase.repos.add(repo)
         self.repoid = repo.id
         try:
-            logger = logging.getLogger('dnf')
-            logger.setLevel(logging.ERROR)
-            self.yumbase.repos[self.repoid].load()
-            logger.setLevel(logging.WARN)
-        except RepoError:
-            # Dnf bug workaround. Mirrorlist was provided but none worked. Fallback to baseurl and load again.
```

Comment:
Which DNF bug do you refer here to? As I personally don't have the context do we need to have a minimum dnf version in the specfile?

Reply:
If was exact a year ago I was wery confused to see the date on the first glance:
https://github.com/uyuni-project/uyuni/commit/6cea1af1cfc4f4df82d3f5951264f6efc71c9b8a

Reply:
@sbluhm which bug you mentioned in the initial commit from a year ago?